test/spire-proxy/main.go

```diff
@@ -0,0 +1,191 @@
+package main
```

Comment:
Is this really a proxy, or are you mocking the spiffe workload API?

Reply:
It is really a proxy, to perform local testing, it starts docker container with spire-server/spire-agent and spire-proxy and allow tests to connect to it via bind TCP port to local machine.

Reply:
Have a look at:
https://github.com/networkservicemesh/cmd-forwarder-vppagent/blob/master/src/forwarder/internal/pkg/cmd/test.go#L51

No need to run a separate container for spire.